Source/DafnyCore/Verifier/BoogieGenerator.ExpressionWellformed.cs

```diff
@@ -1526,17 +1560,17 @@ void CheckWellformedLetExprWithResult(LetExpr e, WFOptions wfOptions, Bpl.Expr r
         var letBody = Substitute(e.Body, null, substMap);
         CheckWellformed(letBody, wfOptions, locals, builder, etran);
         if (e.Constraint_Bounds != null) {
-          var substMap_prime = SetupBoundVarsAsLocals(lhsVars, builder, locals, etran);
-          var nonGhostMap_prime = new Dictionary<IVariable, Expression>();
+          var substMapPrime = SetupBoundVarsAsLocals(lhsVars, builder, locals, etran);
```

Comment:
Really??? I don't think it improves anything. But sure I'll rename it.

Reply:
For me it improves maintainability. At first when I looked at "substMapPrime" I was thinking of prime numbers, which had nothing to do with that. Then I looked everywhere to see what "prime" was referring to and I did not find that. Then I realized it could be a character encoding of the ' and it made sense. Just want to avoid future confusion.
We could also replace the "2" by something more self-explaining, like `substMapLocalVars`